### ISSUE TITLE
The Wowozela won't get listed/detected (new gmod bug?)

### ISSUE THREAD

**Issue by VictorienXP** [Foohy/jazztronauts#86]
So there is this new bug and I think this might entirely be a new Garry's Mod bug and I should probably check/report that up...

But I'll say it here anyway:

I've been debugging to see what's happening, and it seems that weapons.GetList() will basically give nothing when the script gets executed. But it does works when I reload the script manually. So the weapon table might not be initialized for some reason or something.

To know that I've added some print() to thirdparty.lua:
```lua
local weps = weapons.GetList()
local lookup = {}
MsgC(Color(255, 0, 0), "----------------------------------------------------------------------------------------------------------------WEPS\n")
for _, v in pairs(weps) do
	lookup[v.ClassName] = v
	print("WEP", v.ClassName, v.PrintName, v.Folder)
end
```

And so when everything gets initialized we can see there is only "none":
![image](https://user-images.githubusercontent.com/1633366/70663461-aba33d80-1c68-11ea-9b5a-e06755276ad3.png)
![image](https://user-images.githubusercontent.com/1633366/70663170-1011cd00-1c68-11ea-9fd2-eb10b1620c77.png)
![image](https://user-images.githubusercontent.com/1633366/70663965-bb6f5180-1c69-11ea-9195-f865a22569e6.png)
![image](https://user-images.githubusercontent.com/1633366/70663362-7c8ccc00-1c68-11ea-8ae8-d7aeb0159e6a.png)

But yeah when I reload the script, it does work:
![image](https://user-images.githubusercontent.com/1633366/70663518-d42b3780-1c68-11ea-8b01-d3e7a93eddc0.png)
![image](https://user-images.githubusercontent.com/1633366/70663724-397f2880-1c69-11ea-93e3-32256cac856a.png)
![image](https://user-images.githubusercontent.com/1633366/70663615-0472d600-1c69-11ea-8761-7601a1453bb9.png)

I don't know if there is a addon messing with me or if this is a new gmod bug but considering it does work after everything got loaded first that might be a change of behavior with gmod.

So a workaround will be to not call weapons.GetList() too early...

Anyway I'm really not sure on why this is happening now and since when.
Maybe I could retry with a clean gmod just to be sure.
But if someone could check if indeed they got this bug too that would be nice.

**Comment by thekirbykid20067** [comment]
How do I reload the script?

**Comment by VictorienXP** [comment]
> How do I reload the script?

I mean it's [Lua auto refresh](https://wiki.facepunch.com/gmod/Auto_Refresh).

**Comment by Foohy** [comment]
Fixed in 371a46ea2bcf69fb8f3c8541043013c8f267a82e
